docs/SUBWORKFLOWS.md

```diff
@@ -0,0 +1,147 @@
+# Adding a new subworkflow to nf-scil
+
+## Generate the template
+
+First verify you are located at the root of this repository (not in `subworkflows`), then run the following interactive command :
+
+```
+nf-core subworkflows create
+```
+
+It will ask you to give a name for your subworkflow and an author name (use your github username).
+Alternatively, you can use the following command to supply directly those informations :
+
+```
+nf-core subworkflows create <name> --author <author>
+```
+
+## Generate the template
+
+### Edit `./subworkflows/nf-scil/<name_of_your_workflow>/main.nf`
+
+You don’t have the choice to generate an empty template when you generate a subworflow, so the template is based on nf-core.
+
+- Remove the different comment lines.
+- Include your modules into your subworkflows. Remove the modules `{ SAMTOOLS_SORT}` and `{ SAMTOOLS_INDEX }` then includes yours with the good pathway:
+
+```
+include { <MODULES>	} from '../../../modules/nf-scil/<category>/<tool>/main'
+```
+
+> [!NOTE]
+> You can also include other subworkflows :
+>
+> ```
+> include { <SUBWORKFLOW> } from '../<subworkflow>/main'
+> ```
+
+#### Define your Workflow inputs.
```

Comment:
I feel using subworkflow would be more consistent with the rest of the doc, but I don't mind workflow either. I'll let you choose!
```suggestion
#### Define your Subworkflow inputs.
```